packages/block-library/src/comments-pagination-numbers/edit.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * WordPress dependencies
+ */
+import { useBlockProps } from '@wordpress/block-editor';
+
+const PaginationItem = ( { content, tag: Tag = 'a', extraClass = '' } ) => (
+	<Tag className={ `page-numbers ${ extraClass }` }>{ content }</Tag>
+);
+
+export default function QueryPaginationNumbersEdit() {
+	return (
+		<div { ...useBlockProps() }>
+			<PaginationItem content="1" />
```

Comment:
Nitpick: should it use the regular `children` instead of the `content` prop. I guess it doesn't matter but it's shorter as is 🤷

Reply:
I thought the same, but I ended up using self-closing tags because I found it shorter and easier to read, haha. 😅 
```jsx
<>
  <PaginationItem>1</PaginationItem>
  <PaginationItem>2</PaginationItem>
  <PaginationItem tag="span" extraClass="current">
    3
  </PaginationItem>
</>
```

Anyway, I can change it to `children` if you think is better.  😁

Reply:
We can leave it as is. It doesn't matter that much.